docs/source/setup.md

```diff
@@ -231,3 +231,105 @@ Possible solutions for the same problem can be found at [link]
 [link]: http://stackoverflow.com/a/28938258/2534102
 [backend]: http://127.0.0.1:8000
 [frontend]: http://127.0.0.1:8888
+
+## Windows Installation Instructions
+
+### Step 1: Install prerequisites
+
+* Install git from https://git-scm.com/downloads
+
+* Install postgres from https://www.postgresql.org/download/windows/
+
+* Install rabbitmq from https://www.rabbitmq.com/install-windows.html
+Also install supported version of Erlang for windows given in the above link
+
+* Install python v2.7 from https://www.python.org/downloads/
+
+* Install virtualenv
+
+```shell
+pip install virtualenv
+```
+### Step 2: Get EvalAI code
```

Comment:
Break it into two headings like:
1. For users authenticating with SSH key
2. For users authenticating without SSH key

Reply:
Yeah, that'll be helpful. I have already mentioned the steps for non SSH